### PR TITLE
updated the error message for missing features

### LINKED ISSUE #520: Standalone raises incorrect parameter error
@gibsramen 

Hi empress team, 

When using the standalone version, a wrong error message occurs.
i.e.
I tried to run the following, but got an error message indicating that I was missing the --p-filter-missing-features flag. 
!empress community-plot \
    --tree tree.nwk \
    --table 61337_reference-hit.biom \
    --sample-metadata 1070_20170412-111709.txt \
    --output-dir community-tree-viz3

Error message: File "/opt/miniconda3/envs/qiime2-2020.11/lib/python3.6/site-packages/empress/tools.py", line 247, in match_inputs
    "The feature table contains features that aren't present as "
empress.tools.DataMatchingError: The feature table contains features that aren't present as tips in the tree. You can override this error by using the --p-filter-missing-features flag.

The message should reflect the difference between standalone and qiime 2. 

I'd like to try fix this issue if that's possible. Thank you.

### PR DESCRIPTION
**Pull request by ericap258**
This pull request closes #520

### CODE REVIEW

**Comment by kwcantrell** [comment]
Thanks @ericap258 fixing this!

**Comment by kwcantrell** [comment]
@ericap258 would you be able to pull master? We are currently having issues with McHelper so I have temporarily removed it from our CI since it is only used to make reviewing PRs  easier and not used within empress. This PR is using our CI with McHelper and thus the build is failing.

File: empress/tools.py [hunk 1]
```diff
@@ -263,7 +263,9 @@ def match_inputs(
             raise DataMatchingError(
                 "The feature table contains features that aren't present as "
                 "tips in the tree. You can override this error by using the "
-                "--p-filter-missing-features flag."
+                "--p-filter-missing-features flag for QIIME 2 or "
+                "--filter-missing-features flag for the standalone."
+
```
Comment (fedarko) [inline empress/tools.py:268]:
```suggestion
                "--p-filter-missing-features flag for the QIIME 2 version of "
                "EMPress, or the --filter-missing-features flag for the "
                "standalone version of EMPress."
```

Comment (fedarko) [reply empress/tools.py:268]: This is a small suggestion that fixes the wording of the error message slightly. Thank you!

**Comment by fedarko** [comment]
Thanks @ericap258! It looks like there are a few errors from the `flake8` linter that are causing the build to fail. If you wouldn't mind fixing these, we can merge this in afterwards. Thank you!

File: empress/tools.py [hunk 2]
```diff
@@ -263,7 +263,10 @@ def match_inputs(
             raise DataMatchingError(
                 "The feature table contains features that aren't present as "
                 "tips in the tree. You can override this error by using the "
-                "--p-filter-missing-features flag."
+                "--p-filter-missing-features flag for the QIIME 2 version of "
++               "EMPress, or the --filter-missing-features flag for the "
++               "standalone version of EMPress."
```
Comment (ElDeveloper) [inline empress/tools.py:268]: With strings inside a parenthesis, the `+` operators are not needed.

```suggestion
                "EMPress, or the --filter-missing-features flag for the "
                "standalone version of EMPress."
```

**Comment by emperor-helper** [comment]
The following artifacts were built for this PR: [empire-biplot.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F522%2Fempire-biplot.qzv), [empire.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F522%2Fempire.qzv), [empress-tree.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F522%2Fempress-tree.qzv), [just-fm.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F522%2Fjust-fm.qzv), [plain.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F522%2Fplain.qzv)

**Comment by ElDeveloper** [comment]
Thanks so much @ericap258

**Pull request merged**